### PR TITLE
Remove Debug configuration for TestNativeService.

### LINKED ISSUE #12942: Test failure: System.ServiceProcess.Tests.ServiceControllerTests/ConstructWithDisplayName
Opened on behalf of @jiangzeng

The test `System.ServiceProcess.Tests.ServiceControllerTests/ConstructWithDisplayName` has failed.

System.Exception : error: System.ServiceProcess.ServiceController.TestNativeService.exe failed with exit code -1073741515

        Stack Trace:

           at System.ServiceProcess.Tests.ServiceProvider.RunServiceExecutable(String action)
       at System.ServiceProcess.Tests.ServiceProvider.CreateTestServices()
       at System.ServiceProcess.Tests.ServiceProvider..ctor()
       at System.ServiceProcess.Tests.ServiceControllerTests..ctor()
Build : Master - 20161023.01 (Core Tests)
Failing configurations:
- Windows.7.Amd64
  - AnyCPU-Debug
- Windows.10.Core.Amd64
  - AnyCPU-Debug
- Windows.81.Amd64
  - AnyCPU-Debug
- Windows.10.Amd64
  - AnyCPU-Debug

Details:
https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20161023.01/workItem/System.ServiceProcess.ServiceController.Tests/analysis/xunit/System.ServiceProcess.Tests.ServiceControllerTests~2FConstructWithDisplayName

### PR DESCRIPTION
**Pull request by Priya91**
fixes #12942 

cc @karelz @weshaggard

### CODE REVIEW

File: src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestNativeService/System.ServiceProcess.ServiceController.TestNativeService.vcxproj [hunk 1]
```diff
@@ -2,18 +2,6 @@
...
   <ItemGroup Label="ProjectConfigurations">
-    <ProjectConfiguration Include="Debug|Win32">
-      <Configuration>Debug</Configuration>
-      <Platform>Win32</Platform>
-    </ProjectConfiguration>
-    <ProjectConfiguration Include="Debug|x64">
-      <Configuration>Debug</Configuration>
-      <Platform>x64</Platform>
-    </ProjectConfiguration>
-    <ProjectConfiguration Include="Release|Win32">
```
Comment (weshaggard) [inline src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestNativeService/System.ServiceProcess.ServiceController.TestNativeService.vcxproj]: I suspect not but do we have any 32-bit test builds/runs that would need the win32 version?

Comment (Priya91) [inline src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestNativeService/System.ServiceProcess.ServiceController.TestNativeService.vcxproj]: I don't think so, if this breaks, can bring it back.

**Comment by Priya91** [comment]
@dotnet-bot test Innerloop Ubuntu14.04 Debug Build and Test

**Comment by Priya91** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test

**Comment by Priya91** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test

**Comment by Priya91** [comment]
Windows CI failing with test failure issue #13025

**Pull request merged**